src/main/entities/Event.java

```diff
@@ -0,0 +1,146 @@
+package main.entities;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * A class that represents an event at the conference.
+ *
+ * @author Yile Xie
+ */
+
+public class Event {
+    private final UUID id = UUID.randomUUID();
+    private String title;
+    private LocalDateTime time;
```

Comment:
@YvetteXie LocalDateTime or Date? I'm not sure which one is better

Reply:
Date has a simpler format, but since we have to specify the hours of each event (for scheduling purposes), if we choose to use Date, then we need another variable to indicate the hours. I just thought that would be more work for us.